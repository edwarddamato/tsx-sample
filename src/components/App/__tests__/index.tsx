jest.mock('../../../api/github');

import * as React from 'react';
import { App } from '../index';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const app = create(
    <App />
  ).toJSON();
  expect(app).toMatchSnapshot();
});