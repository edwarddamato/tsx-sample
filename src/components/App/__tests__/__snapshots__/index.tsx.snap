// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="root_container"
>
  <h1>
    Let's get a Github user
  </h1>
</div>
`;
